doozer/doozerlib/cli/rpms.py

```diff
@@ -302,3 +311,58 @@ async def _build_rpm(runtime: Runtime, builder: RPMBuilder, rpm: RPMMetadata):
             record["task_url"] = task_urls[0]
         runtime.record_logger.add_record(action, **record)
     return record["status"]
+
+
+async def update_konflux_db(runtime, rpm: RPMMetadata, record: dict):
+    nvrs = record["nvrs"].split(",")
+
+    with runtime.shared_koji_client_session() as koji_api:
+        builds = get_build_objects(nvrs, koji_api)
+
+    _, version, _ = await exectools.cmd_gather_async(
+        ["rpmspec", "-q", "--qf", "%{version}", "--srpm", "--undefine", "dist", "--",
+         str(rpm.specfile)])
+
+    for build in builds:
+        rebase_url = build["extra"]["source"]["original_url"].split('+')[-1]
+        rebase_repo_url, rebase_commitish = rebase_url.split('#')
+
+        with open(rpm.specfile) as specfile:
+            rpmspec = specfile.read().splitlines()
+            try:
+                commitish = [line for line in rpmspec if "Update to source commit" in line][0].split(" ")[-1].split("/")[-1]
+            except IndexError:
+                rpm.logger.warning('Could not determine commitish for rpm %s', rpm.rpm_name)
+                commitish = ''
+
+        build_record = KonfluxBuildRecord(
+            name=rpm.rpm_name,
+            group=runtime.group,
+            version=version,
+            release=rpm.release,
+            assembly=runtime.assembly,
+            el_target=build["nvr"].split(".")[-1],
```

Comment:
I think we can use `isolate_el_version_in_release` here

Reply:
Agreed, done in eb4c0541ab0194d3d2c109b34250ab7a44a4c7b0